src/store/includes-labels.ts

```diff
@@ -125,10 +135,12 @@ export async function getPrice(
   const priceString = await extractPageContents(page, selector);
 
   if (priceString) {
-    const price = Number.parseFloat(
-      priceString.replace(/\\.|\\,/g, '').match(/\d+/g)!.join('.') // eslint-disable-line
-    );
-
+    const euroFormat = priceString.indexOf('.') < priceString.indexOf(',');
+    let price = getPriceFromString(priceString, euroFormat);
+    //Sanity check, flip "euroFormat" if price lower than 50
+    if (price < 50) {
+      price = Math.max(price, getPriceFromString(priceString, !euroFormat));
+    }
```

Comment:
```suggestion
    let price = getPriceFromString(priceString);
```

Reply:
The "Sanity check" is now removed in fc3c32f and so separate function is no longer needed.